integrationtests/osgi/src/test/java/org/infinispan/it/osgi/util/IspnKarafOptions.java

```diff
@@ -332,6 +332,7 @@ public static Option featurePaxUrlWrap() throws Exception {
    public static Option commonOptions() throws Exception {
       return composite(karafContainer(),
                        vmOptions("-Djava.net.preferIPv4Stack=true", "-Djgroups.bind_addr=127.0.0.1"),
+                       vmOptions("-Xmx1G", "-XX:-PrintGCDetails", "-XX:-PrintGCTimeStamps"),
```

Comment:
hmmm it looks a bit weird - we set Xmx in `integrationtests/osgi/pom.xml` and here we use 1GB. Could you please tell me why? And shouldn't this be parametrized (e.g. taken from pom)?

Reply:
There are 3 JVMs running:

1. The Maven JVM - uses heap size from `$MAVEN_OPTS` or 1/4 or RAM.
2. The Surefire fork - uses heap size from `forkJvmArgs` (2G in `parent/pom.xml`), `$MAVEN_FORK_OPTS`, or 1/4 of RAM if `$MAVEN_FORK_OPTS` is non-empty and doesn't have `-Xmx`.
3. The Karaf process - 1/4 of RAM.

In this PR, I'm experimenting with the heap size of all 3 JVMs. I could use the same heap size for VMs 2 and 3, but I don't think it's ok to give that much memory to the Surefire fork when all it does is start the Karaf process.